cedar-policy-core/src/entities.rs

```diff
@@ -893,17 +1067,16 @@ mod json_parsing_tests {
 
     #[test]
     fn implicit_and_explicit_escapes() {
-        // this one tests the implicit and explicit forms of `__expr` and `__entity` escapes
+        // this one tests the implicit and explicit forms of `__entity` escapes
```

Comment:
it's nice that we adapted this test to still check all the implicit/explicit forms we do allow.  Can we also add a test for sane errors when we use implicit and/or explicit forms of `__expr`?  (Maybe you did, I just don't see it in this file if that's the case)

Reply:
Good call out